source/v4/system.yaml

```diff
@@ -14,3 +14,20 @@
             $ref: "#/definitions/StatusOK"
         '400':
           $ref: '#/responses/BadRequest'
+
+  '/database/recycle':
+    post:
+      tags:
+        - system
+      summary: Recycle the database connection
```

Comment:
Can the summary be `Recycle database connections` and the description be `Recycle database connections by closing and reconnecting all connections to master and read replica databases.`